.travis.yml

```diff
@@ -48,26 +62,9 @@ after_success:
           --exclude "test/test-subsystem"
           --exclude "bindings/c/Test.cpp"
           --exclude "test/tokenizer"
+          --gcov /usr/bin/gcov-4.8
           --gcov-options '\--long-file-names --preserve-paths'
 
 notifications:
-  email:
-    - david.wagner@intel.com
```

Comment:
see commit message... but you are right, in theory I should have done it in a separate patch.....